pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -1372,11 +1373,18 @@ public static String getSLAMonitorBrokerName(ServiceUnitId ns) {
     }
 
     public static boolean isSystemServiceNamespace(String namespace) {
-        return HEARTBEAT_NAMESPACE_PATTERN.matcher(namespace).matches()
+        return SYSTEM_NAMESPACE.toString().equals(namespace)
+                || HEARTBEAT_NAMESPACE_PATTERN.matcher(namespace).matches()
```

Comment:
may can use isHeartbeatNamespace()

Reply:
yes, right. Fixed.

Reply:
The test : OverloadShedderTest#testBrokerWithMultipleBundles could not pass. I change back to logic here.